docs/concepts/registries.md

```diff
@@ -40,63 +28,133 @@ public ExampleMod() {
 }
 ```
 
-Creating Registries
+### `Register` events
+
+The `RegistryEvent`s are the second and more flexible way to register objects. These [events][] are fired after the mod constructors and before the loading of configs.
+
+The event used in registering objects is the `RegistryEvent.Register<T>`. The type parameter `T` should be set to the type of the object being registered. Calling `#getRegistry` will return the registry, upon which objects are registered with `#register` or `#registerAll`. 
+
+`RegistryEvent.Register<?>` events are fired in this order: first, the `Block` registry, then the `Item` registry, and then all other registries in alphabetical order. 
+
+Here is an example: (note that the event handler must be registered on the *mod event bus* in the mod constructor)
```

Comment:
```suggestion
Here is an example: (The RegistryEvents must be added to the *mod event bus*)
```
Not sure about my phrasing, but yours seemed a bit subjective.

Reply:
Reading that, it does seem... off to me.
```suggestion
Here is an example: (the event handler is registered on the *mod event bus*)
```
Might that work instead?

Reply:
sure